bc/subscription/templates/includes/search_htmx/case-form.html

```diff
@@ -35,20 +35,21 @@
       </div>
       {% for group in channels %}
         <div class="flex flex-col text-base">
-          <div class="py-2 bg-gray-50">
-            <label for="id_group_{{group.pk}}" class="flex items-center mx-2">
-              <input type="checkbox" name="groups" value="{{group.pk}}" id="id_group_{{group.pk}}" checked data-cg-target="group-{{group.pk}}">
-              <div class="flex ml-2 font-medium">
-                {{ group.name }}
-              </div>
-            </label>
+          <div class="p-2 bg-gray-50 flex justify-between font-medium">
+            <div>
+              {{ group.name }}
+            </div>
+            <div class="text-sm self-end">
+              <a href="javascript:void(0)" class="underline text-blue-600 hover:text-blue-800" data-cg-target="group-{{group.pk}}" data-cg-uncheck="check-none-group-{{group.pk}}">All</a> /
+              <a href="javascript:void(0)" class="underline text-blue-600 hover:text-blue-800" id="check-none-group-{{group.pk}}">None</a>
```

Comment:
Is there a reason to do these as links instead of as spans that look like links?